x-pack/plugins/lens/public/drag_drop/drag_drop.tsx

```diff
@@ -247,4 +267,188 @@ const DragDropInner = React.memo(function DragDropInner(
     onDragEnd: dragEnd,
     onDragStart: dragStart,
   });
+
+  if (
+    droppable &&
+    dropType === 'reorder' &&
+    itemsInGroup?.length &&
+    itemsInGroup.length > 1 &&
+    value?.id
+  ) {
+    element = (
+      <ReorderableDragDrop
+        draggingProps={{
+          className: classNames(children.props.className, classes),
+          draggable,
+          onDragEnd: dragEnd,
+          onDragStart: dragStart,
+          ['data-test-subj']: props['data-test-subj'] || 'lnsDragDrop',
+        }}
+        dropProps={{
+          onDrop: drop,
+          onDragOver: dragOver,
+          onDragLeave: dragLeave,
+          dragging,
+          droppable,
+          itemsInGroup,
+          id: value.id,
+        }}
+      >
+        {children}
+      </ReorderableDragDrop>
+    );
+  }
+
+  if (!itemsInGroup?.length || itemsInGroup.length < 2 || !value || !dropTo || !draggable) {
+    return element;
+  }
+  const groupLength = itemsInGroup.length;
+  const currentIndex = itemsInGroup.indexOf(value.id);
+
+  const { label } = props as DraggableProps;
+
+  return (
+    <>
+      <EuiScreenReaderOnly>
```

Comment:
Would appreciate some help testing this!

Reply:
I've added some description around accessibility, let me know if you need more help – but I am waiting for possible design changes so let's maybe hold on with testing this till the design is final.